lib/gollum-lib/file.rb

```diff
@@ -145,15 +147,21 @@ def get_disk_reference(name, commit)
     # that if you specify try_on_disk=true, you may or may not get a file
     # for which on_disk? is actually true.
     def find(name, version, try_on_disk = false)
-      checked = name.downcase
       map     = @wiki.tree_map_for(version)
+      if @wiki.page_file_dir
+       path = ::File.join(@wiki.page_file_dir, name)
+      else
+       path = name
+      end
+      query = ::File.join('/', path)
```

Comment:
Perhaps we could define a constant `PATH_SEPARATOR='/'` and use that throughout the code base instead of sprinking `'/'` where needed?

Reply:
There is ::File::SEPARATOR, which I'll use!

Reply:
Perfect. 👍

Reply:
Hmm, I'm not sure now what the right approach is. Seeing as the generated paths will end up being used as the page or file's `url_path`, we really don't want anything other than slashes (backslashes?) popping up there. According to [this](https://www.ruby-forum.com/t/path-separator-and-windows/51339), slash should work even on Windows. I've reinstated the slashes for now, but let me know what you think is the best approach!